CMakeLists.txt

```diff
@@ -3,7 +3,7 @@ cmake_minimum_required(VERSION 3.15)
 cmake_policy(SET CMP0091 NEW) # enable new "MSVC runtime library selection" (https://cmake.org/cmake/help/latest/variable/CMAKE_MSVC_RUNTIME_LIBRARY.html)
 
 project(libCZI 
-      VERSION 0.49.0
+      VERSION 0.49.1
```

Comment:
Isn't this a backward compatible change rather than a bugfix?

Reply:
You mean we should increment the MINOR version? Ja, I guess... you're right.

Reply:
done, thanks